apps/system/test/marionette/statusbar_home_button_test.js

```diff
@@ -52,6 +52,8 @@ marionette('Closing statusbar via home button >', function() {
            .release()
            .perform();
 
+    client.helper.wait(2000);
```

Comment:
We need to figure this one out. We wouldn't let a random wait land as part of a small patch we can't let it slip here.